backend/typescript/graphql/types/authType.ts

```diff
@@ -0,0 +1,12 @@
+import { gql } from "apollo-server-express";
+
+const authType = gql`
+  extend type Mutation {
+    login(email: String!, password: String!): String!
```

Comment:
Possible improvement, could return a Response type that contains an "ok" field, so on frontend we can check if mutation succeeded or not
Not sure if this applies though if there won't be any errors

Reply:
My thought was to have the client do an error check by seeing if the `errors` field is present in the response, not sure if this is the usual practice for GraphQL though so let me know!

Reply:
Oh ok, I didn't know that we had the errors field, that should be good then